python/lsst/afw/fits.cc

```diff
@@ -22,20 +22,28 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
```

Comment:
Sounds reasonable (especially since we do plan to take a final cleanup pass when pybind11 porting is complete). Unlike some commented-out code it's pretty obvious what this does.